src/cargo/util/toml/mod.rs

```diff
@@ -164,6 +170,16 @@ pub fn parse(toml: &str, _file: &Path, _config: &Config) -> CargoResult<toml::Va
         .map_err(|e| anyhow::Error::from(e).context("could not parse input as TOML"))
 }
 
+pub fn parse_document(
```

Comment:
Two questions.
1. Is the old `parse` dead code now?
2. Does this pull its weight as a standalone function at this point, or can it be inlined into where it is used?

Reply:
`parse` is still used in 3 places.

`parse_document` is used in 1 place and was intended as a parallel to it when a `toml_edit::Document` was needed instead of a `toml_edit::easy::Value`.

The main benefit I see to keeping both functions and keeping them together is to have a consistent error message.  Especially if we ever intended to use the parameters that are currently ignored.

That said, I'm fine removing the functions or keeping them/